Assessment/Documentation/Documentation.md

```diff
@@ -6,7 +6,7 @@ In the same directory tree as this document is a folder named Wireframes, there
 ## 1.1.3 Color Choices
 The color scheme for this website was easy to choose, as the Instagram account my partner uses for her hair is called “Jadie Lavender”. So, it only makes sense to choose a color scheme with lavender or light purple in. At first, I used a block color throughout the web pages, but this made the website seem outdated. I then had the idea to use a slight gradient in the background color to give it a more modern feel. I used the base color of her Logo as a starting point. Using a hex color picker I slightly lightened the base color and set that as the second color. I lightened the secondary color again to give a tertiary color which would go in the middle of the web pages. The gradient starts dark, gets a little lighter, gets more lighter, goes back to darker and finishes on the same color as the Logo. Paired with a white font color, I think this color scheme gives an elegant and refined look to the web pages that doesn’t detract from the content.
 ## 1.1.4 Accessibility 
-To conform with W3C, I used their Nu Html Checker (found here - https://validator.w3.org/ ) this goes through the html of a webpage and spots any errors, I used the upload file option. The Home page only had a few issues, some ids needed to be changed to classes and there were comments that needed taken out. I also got a colleague (who is a web developer turned software engineer) to code review it.
+To conform with W3C, I used their Nu Html Checker (found here - https://validator.w3.org/ ) this goes through the html of a webpage and spots any errors, I used the upload file option. The Home page only had a few issues, some ids needed to be changed to classes and there were comments that needed taken out. I also got a colleague (who is a web developer turned software engineer) to code review it. More information and screenshots can be found at Documentation/W3C_Compliance_Evidence.
 ## 1.1.5 GDPR
```

Comment:
'if this were a webaite' - typo
'hire a lawyer' - americanisation, for UK use 'seek legal advice'
'format the provacy' - typo
'information concerining the 8 individual rights' - typo and remove '8' as this is likely to cause future issues for this readme if the amount changes
---
Observation only - Is an email the best methodology? While the requirement is to ensure you have delivered the information, it's easier to show it with a very simple and big 'close' button for those that don't care?

Reply:
'learnt' where should be 'learned'
'My design chose was directed by my partner,' should be 'choice'